components/chip/chip.vue

```diff
@@ -145,6 +145,14 @@ export default {
       type: [String, Array, Object],
       default: '',
     },
+
+    /**
+     * Additional class name for the span element.
+     */
+    parentClass: {
+      type: [String, Array, Object],
+      default: '',
+    },
```

Comment:
You shouldn't need to do this, if you pass something in as just `class="whateverclass"` it will automatically be merged with the classes on the parent element.

https://v2.vuejs.org/v2/guide/class-and-style.html#With-Components

Reply:
Hmm you are definitely right that vue 3 binds class with `v-bind="$attrs"` while vue 2 does not.

I think we originally made a bit of a mistake in setting `v-bind="$attrs"` on the inner label here. It doesn't really make sense that it would set it on that element by default. I'd prefer if we had no `v-bind="$attrs"` at all set and then had a prop "labelClass" that would set classes on that element.

This is somewhat of a breaking change but there's not much usage of chip so I will check if it broke anything with the current product implementations after release

Reply:
I agree on this `I'd prefer if we had no v-bind="$attrs"` but idk how much you guys want to have similitude between v2 and v3, thats why I didn't clean or touch anything

`then had a prop "labelClass"` Okay, I'll update `parentClass` to `labelClass `

Reply:
After we make this change it should be the same in vue 2 / vue 3 again. But in general if we have to change vue 3 branch because of a breaking change between vue 2 and vue 3, it is fine to make that change.